.github/workflows/docker.yml

```diff
@@ -0,0 +1,42 @@
+name: Docker
+
+on:
+  push:
+    branches:
+      - 'main'
+  release:
+    types: [published]
+
+jobs:
+  docker:
```

Comment:
we need another job where we push images from other docker processes, those we would push only if there are changes to the docker dir, example the exposer.
These will be more of a one time thing, but we just tag them with commit itself too

Reply:
For this I think we should create a new action for this then. If we use both the branches filter and the paths filter, the workflow will only run when both filters are satisfied.

Reply:
i would not want to create another yaml for docker. all docker should be from the same i feel